packages/ag-charts-community/src/api/agChart.ts

```diff
@@ -67,6 +63,9 @@ export abstract class AgCharts {
         this.licenseChecked = true;
     }
 
+    /** @private - for use by Charts website dark-mode support. */
+    static optionsMutationFn?: (opts: AgChartOptions) => AgChartOptions;
```

Comment:
We could probably just monkey patch the prototypes more - but can revisit that later